### PR TITLE
Nova configuration and new location

### PR DESCRIPTION
**Pull request by aplanas**

### CODE REVIEW

**Comment by aplanas** [comment]
[(5) Update templates for config files for Newton - Nova](https://trello.com/c/qbXXTDIE/379-5-update-templates-for-config-files-for-newton-nova)

**Comment by toabctl** [comment]
@aplanas nova-api does not start:
```
2016-12-19 16:53:14.708 1839 ERROR nova MissingRequiredOptions: Auth plugin requires parameters which were not given: auth_url

```

**Comment by aplanas** [comment]
@toabctl actually the fail is because of some crudini bits in /etc/nova/nova.conf that comes from the package. I guess that when the changes in the package are merged, and the crudini info is not there anymore, this will pass (I tested)

**Comment by aplanas** [comment]
* Remove some blank lines and expand some `if`. Still generate the same config file

**Comment by rsalevsky** [comment]
Needs a rebase

File: chef/cookbooks/nova/attributes/default.rb [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
 default[:nova][:debug] = false
 default[:nova][:max_header_line] = 16384
+default[:nova][:config_file] = "/etc/nova/nova.conf.d/100-nova.conf"
```
Comment (jsuchome) [inline chef/cookbooks/nova/attributes/default.rb]: Is this correct? My installed file is `010-nova.conf`

**Comment by toabctl** [comment]
Taking over from @aplanas with https://github.com/crowbar/crowbar-openstack/pull/697

**Pull request closed without merging**